src/components/todo.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import TodoList from "./todolist";
+// your components will all go in this `component` directory.
+// feel free to change this component.js into TodoList.js
+
+class Todo extends Component {
+    constructor() {
+        super();
+        this.state = {
+        todos: "",
+        todo: "",
+        
+        }
+    };
+
+
+    nameReciever = (event) => {
+        this.setState({ [event.target.description]: event.target.value });
+    }; 
+    
+    handleSunmitTodo = () => {
+        const todos = this.state.todos;
+        const todo = { name: this.state.todo, id:this.state.todo };
+        todos.push(todo);
+        this.setState({ todos: todos, todo: "" });
+    };
+
+
+
+    render() {
+        return (
+        <div>
+         Todo 
+          <todos todo={this.state.todo} />   
+            <input name="todos" onChange={this.nameReciever} value={this.state.todos} placeholder="React palyground" />
```

Comment:
The input and button belong inside the TodoList class, and not inside the Todo. Inside the TodoList you should instead pass the information that the user inputs as props to each Todo being created.